src/dune_apis.rs

```diff
@@ -176,25 +139,28 @@ mod tests {
     use super::*;
 
     #[tokio::test]
-    async fn test_get_function_name() {
+    async fn test_get_function_name() -> Result<()> {
         let client = reqwest::Client::new();
         let function_signature = "0x70a08231".to_owned();
         println!("Function signature: {}", function_signature);
         let function_name =
-            get_function_name(&client, QUERY_FUNCTION_NAME_SIGNATURE, function_signature).await;
-        println!("Function name:      {}", function_name);
+            get_function_name(&client, QUERY_FUNCTION_NAME_SIGNATURE, function_signature).await?;
+        println!("Function name:      {:?}", function_name);
```

Comment:
This was actually `{}` on purpose because we just want the name without the `""` (which also gets escaped, making it a bit unreadable).

But it's just a debug print anyway so not a big deal.